src/index.js

```diff
@@ -1,28 +1,18 @@
-import db from './notifications/db';
-import Configuration from './notifications/config';
-import notificationController from './notifications/controller';
+import NotificationController from './notifications/controller';
 
 class Notification {
-  constructor(config) {
-    this.config = new Configuration(config);
-    this.configDatabase = this.config.database;
-    this.baseApi = this.config.baseApi;
+  constructor(database) {
+    this.notificationController = new NotificationController(database);
+
+    this.getAllUnread = this.notificationController.getAllUnread.bind(this.notificationController);
```

Comment:
Yeah it seems to me that is it not being used either. This had to do when I tried to use the class `MongoNotificationDatabase` that had a connection to mongo on the constructor. But that's not there anymore so I should remove this, thank you!

And in fact all of these methods seems unused to me